rotkehlchen/db/schema.py

```diff
@@ -724,19 +724,19 @@
         WHERE first1in > 0 AND first0in > 0 AND last1out > 0 AND last0out > 0
     ), SWAPS AS (
     SELECT base1 AS base_asset, quote1 AS quote_asset, amount1 AS amount, rate1 AS rate, txhash, logindex, time, location FROM C1
-    UNION
+    UNION ALL /* using union all as there can be no duplicates so no need to handle them */
     SELECT base1 AS base_asset, quote1 AS quote_asset, amount1 AS amount, rate1 AS rate, txhash, logindex, time, location FROM C2
-    UNION
+    UNION ALL /* using union all as there can be no duplicates so no need to handle them */
```

Comment:
Does `UNION ALL` perform better than only `UNION` in the case of no duplicates? Couldn't find anything in the docs here https://www.sqlite.org/optoverview.html

Reply:
UNION is a lot more expensive an operation than UNION ALL as it also sorts unioned table and removes duplicates.

Reply:
https://stackoverflow.com/a/3627970/110395